<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dictionary</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>771</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListWidget" name="englishList"/>
     </item>
     <item>
      <widget class="QListWidget" name="russianList"/>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>English:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="englishEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Russian:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="russianEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>100</y>
          <width>231</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="addPair">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>140</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Add pair</string>
        </property>
       </widget>
       <widget class="QPushButton" name="removePair">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>170</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Remove pair</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveFile">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>140</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Save file</string>
        </property>
       </widget>
       <widget class="QPushButton" name="removeFile">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>170</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Remove file</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>771</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="maximumSize">
          <size>
           <width>525</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Refresh stats</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="statsList">
         <property name="maximumSize">
          <size>
           <width>525</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <widget class="QPushButton" name="playButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>221</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Play game</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="englishField">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="russianField">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>60</y>
          <width>101</width>
          <height>20</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="nextButton">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="results">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>221</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Right answers:</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>addPair</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>622</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveFile</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>714</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>787</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removePair</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeFile</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>790</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>showStats()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>787</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>startGame()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>791</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nextButton</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>getAnswer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>717</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>790</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>buttonPressed()</slot>
  <slot>showStats()</slot>
  <slot>startGame()</slot>
  <slot>getAnswer()</slot>
 </slots>
</ui>
